lib/group_domain_models/lib/src/models/group_info.dart

```diff
@@ -13,7 +13,16 @@ class GroupInfo {
   final String id;
   final String? name, abbreviation;
   final Design design;
-  final String? sharecode, joinLink;
+  final String? sharecode;
+  // Originally we used a joinLink from the backend, but since Firebase Dynamic
+  // Links was deprecated, we needed a quick solution for new links.
+  // Regenerating all existing join links in the database is not feasible right
+  // now, so we compute the joinLink from the sharecode (before we used another
+  // link format).
+  String? get joinLink =>
+      sharecode == null
+          ? null
+          : "https://web.sharezone.net/link?type=joinbykey&data=$sharecode";
```

Comment:
I think a positive side effect of this is that the `joinLink` uses the personal Sharecode now, right? IIRC, used everyone the same `joinLink`